sample_factory/utils/wandb_utils.py

```diff
@@ -55,6 +55,14 @@ def init_wandb_func():
 
     wandb.config.update(cfg, allow_val_change=True)
 
+    wandb.define_metric("train/env_steps")
```

Comment:
This can probably be a loop over a list of patterns to make the code a bit cleaner